lib/Template/gramma.peg

```diff
@@ -130,7 +130,7 @@ ExpressionPath = name:ExpressionName {
     };
 }
 
-ExpressionName = (!'else' name:[@_a-zA-Z_0-9.]+) { return name.join('') }
+ExpressionName = (!'else' name:[@_a-zA-Z_0-9.]+ ) { return text() }
```

Comment:
the named var `name:` isn't used now. i think you can remove it. 
the space after `+` is not necessary